### ISSUE TITLE
Incorrect link for unexpected errors

### ISSUE THREAD

**Issue by fredliporace** [sparkgeo/stac-validator#32]
@jbants congrats on the tool!

minor error, the "unexpected error message" is linking to a incorrect github repo.

```
An unexpected error occurred. Please try again. 
If the problem persists, you can report errors on our Github repo. 
https://github.com/sparkgeo/staclint
```
